tests/gtest/aviftest_helpers.h

```diff
@@ -54,13 +54,20 @@ void FillImageGradient(avifImage* image);
 void FillImageChannel(avifRGBImage* image, uint32_t channel_offset,
                       uint32_t value);
 
-// Returns true if both images have the same features and pixel values.
-// If ignore_alpha is true, the alpha channel is not taken into account in the
-// comparison.
+// Returns true if both arrays are empty or have the same length and bytes.
+// data1 or data2 may be null only when data1_length is 0.
```

Comment:
I originally suggested this comment right before line 164 in aviftest_helpers.cc, where we know `data1_length` is equal to `data2_length`. If we add the comment to this file, we cannot assume they are equal. So the comment should read something like this:

```
// data1 may be null only when data1_length is 0. data2 may be null only when data2_length is 0.
```